build.sbt

```diff
@@ -21,7 +21,7 @@ lazy val common = commonRootSettings ++ Seq(
         Seq("-Xfatal-warnings", "-deprecation:false")
       case Some((2, scalaMinor)) if scalaMinor < 11 =>
         // For scala versions 2.10.x
-        Seq("-Xfatal-warnings")
+        Seq()
```

Comment:
Unfortunately, I have some deprecation warnings. @jonhlee, would you be okay if I added this back and broke backwards compatibility?